src/UI/pages/_app.tsx

```diff
@@ -20,6 +20,7 @@ function MyApp({ Component, pageProps }: AppProps) {
     store.dispatch(getMapStyles());
     store.dispatch(getUiVersion());
     store.dispatch(getApiVersion());
+    store.dispatch(getTileServerOverlays());
```

Comment:
I don't think we want any map-related data loading here, as it will load when any page is rendered, not just the map page. Can you move the tile server and map styles loading to the map page please?

Reply:
Hmm, unless we want to load data behind the scenes, to reduce the loading time for the map page... Any thoughts @ray-millward-tessella ?

Reply:
I'm not really fussed at the moment - ideally I'd like this to all go into one request loaded at the start or we're going to end up with 50 initial requests. There should just be a `getConfig` call but we can change that later.

As long as the call is asynchronous then it will happen in the background - it's just whether it's blocking other calls. We can easily optimise this later.